packages/blaze-elements/package.json

```diff
@@ -33,7 +33,7 @@
     "@blaze-elements/collapsible": "^1.0.0",
     "@blaze-elements/common": "^1.0.0",
     "@blaze-elements/drawer": "^1.0.0",
-    "@blaze-elements/input": "^1.0.0",
+    "@blaze-elements/input": "^2.0.0",
```

Comment:
I did release 2.0.1, because there was already 2.0.0 in npm, I don't see any related changes in git, so we most probably never find our.